Update fondant install command in the readme

mrchtr opened the pull request:
Based on a discussion on discord, we should update the fondant install command in our readme.

RobbeSneyders commented:
Thanks @mrchtr

The code example is outdated as well though. Could you update it?

mrchtr commented:
> Thanks @mrchtr
> 
> The code example is outdated as well though. Could you update it?

I've updated the code example and removed the custom component description (link to the readme now).
